app/Keycloak/Collection/RealmCollection.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Keycloak\Collection;
+
+use App\Keycloak\Dto\Realm;
+use Ramsey\Collection\AbstractCollection;
+
+/**
+ * @extends AbstractCollection<Realm>
+ */
+final class RealmCollection extends AbstractCollection
```

Comment:
Personally I find Ramsey collection better because it actually checks the type on runtime, and is not just a docblock.
But given that we are in Laravel I changed it.